backends/gstreamer/lib.rs

```diff
@@ -73,6 +73,7 @@ impl GStreamerBackend {
         // See https://github.com/servo/media/pull/393.
         let needs_background_glib_main_loop = {
             let (major, minor, micro, _) = gst::version();
+            println!("glib version {major} {minor} {micro}");
```

Comment:
@simonwuelker, is this `println!`  here deliberate or should it be an `info!` or `debug!` invocation instead? I noticed "glibc version 1 24 7" being printed to stdout after updating my main branch and I was confused as it used to be silent before.

Reply:
Oops sorry, that is not intentional :grimacing:

Reply:
https://github.com/servo/media/pull/432